src/main/dynamo/Beyonce.ts

```diff
@@ -144,40 +144,30 @@ export class Beyonce {
     keys: T[]
     consistentRead?: boolean
   }): Promise<GroupedModels<ExtractKeyType<T>>> {
-    const useConsistentRead = params.consistentRead !== undefined ? params.consistentRead : this.consistentReads
```

Comment:
We extract the code that does a single batchGet call into a private method, so it's easier to "chunk" N single calls.